src/pages/PageOrgchart.vue

```diff
@@ -198,8 +200,8 @@ export default {
   async created() {
     window.addEventListener('resize', this.updateView);
     this.updateView();
-    await this.fetchData();
-    await this.highlight();
+    this.highlight();
```

Comment:
if we run `highlight` before `fetchData` and don't wait on it highlight won't work if the orgchart wasn't loaded before, and if it works it's just a lucky won race condition.